src/pages/journal/__tests__/journal.specs.ts

```diff
@@ -1,9 +1,11 @@
 import { ComponentFixture, async, TestBed } from '@angular/core/testing';
```

Comment:
Could you rename the file extension from `specs.ts` to `spec.ts` please? Don't think the test runner will pick it up otherwise.

Reply:
I'll rename this now (although the tests do run).

Reply:
actually - quite a few of the tests had specs.ts will rename all to spec.ts for consistency